package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -1030,11 +1031,12 @@ def _get_timestep():
     @staticmethod
     def calc_angle(d, h, a):
         """Calculate the angle (in degrees) between two atoms with H at apex."""
-        v1 = h.position - d.position
-        v2 = h.position - a.position
-        if np.all(v1 == v2):
-            return 0.0
-        return np.rad2deg(angle(v1, v2))
+        v1= d.position.reshape(1,len(d.position))
```

Comment:
So to cast the array to be 2 dimensional you can do `v1[None, :]` which is a weird numpy trick to add a dimension